.travis.yml

```diff
@@ -23,8 +23,7 @@ cache:
   - "$HOME/.ivy2"
 before_install:
   - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
-      openssl aes-256-cbc -K $encrypted_8d91199f8619_key -iv $encrypted_8d91199f8619_iv -in secret_key.asc.enc -out secret_key.asc -d;
-      gpg --import --batch --passphrase "$PGP_PASSPHRASE" secret_key.asc;
+      gpg --import --batch --passphrase "$PGP_PASSPHRASE" secret_key.asc.enc;
```

Comment:
didn't know this worked with an encrypted key :thinking:

Reply:
Yeah I don't think this is going to work. You definitely need to decrypt the private key before you try to use it with gpg.

@jdesiloniz what kind of errors are you seeing? What are you trying to fix?